app/services/cities_services.py

```diff
@@ -27,14 +28,19 @@ def get_cities_from_state(state):
 
     base_query: Query = session.query(StateModel)
 
-    state = base_query.filter_by(name=state).first()
+    states = base_query.all()
 
-    if not state:
-        raise StateNotFoundError
+    city_state = [
```

Comment:
Since this is a state dict with a list of cities, the name could be state_and_cities, or something like this.